Mods/CombatExtended/Languages/Russian/DefInjected/ThingDef/Buildings_Production.xml

```diff
@@ -1,30 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <LanguageData>
 
-  <!-- Work tables -->
+	<!-- Work tables -->
 
-  <!-- EN: loading bench -->
-  <!--  <AmmoBench.label>верстак боеприпасов</AmmoBench.label>-->
-  <!-- EN: A workstation for handloading ammo. -->
-  <!--  <AmmoBench.description>Верстак для создания боеприпасов. Можно также производить серу из железной руды, порох и напалм. За ним можно работать без подключения к электричеству (40%), но с подключением к сети скорость работы значительно повышается (120%).\nПотребляемая мощность:\nВ простое: 50 W\nПри работе: 500 W</AmmoBench.description>-->
-  <!-- EN: loading bench (blueprint) -->
+	<AmmoBench.label>верстак снаряжения</AmmoBench.label>
+	<AmmoBench.description>Рабочий верстак для создания различных боеприпасов.</AmmoBench.description>
+	<AmmoBench_Blueprint.label>верстак снаряжения (проект)</AmmoBench_Blueprint.label>
+	<AmmoBench_Blueprint_Install.label>верстак снаряжения (проект)</AmmoBench_Blueprint_Install.label>
+	<AmmoBench_Frame.label>верстак снаряжения (постройка)</AmmoBench_Frame.label>
+	<AmmoBench_Frame.description>Рабочий верстак для создания различных боеприпасов.</AmmoBench_Frame.description>
 
-
-
-  <AmmoBench_Blueprint.label>верстак боеприпасов (проект)</AmmoBench_Blueprint.label>
-  <!-- EN: loading bench (blueprint) -->
-  <AmmoBench_Blueprint_Install.label>верстак боеприпасов (проект)</AmmoBench_Blueprint_Install.label>
-  <!-- EN: loading bench (building) -->
-  <AmmoBench_Frame.label>верстак боеприпасов (строительство)</AmmoBench_Frame.label>
-  <!-- EN: A workstation for handloading ammo. -->
-  <AmmoBench_Frame.description>Верстак для создания боеприпасов.</AmmoBench_Frame.description>
-
-  <!-- EN: gunsmithing bench -->
-  <GunsmithingBench.label>оружейный верстак</GunsmithingBench.label>
-  <!-- EN: A workstation for creating and attaching attachments to weapons. -->
-  <GunsmithingBench.description>Оружейный верстак для создания и прикрепления модулей к оружию.</GunsmithingBench.description>
-  <!-- EN: gunsmithing bench (blueprint) -->
-  <GunsmithingBench_Blueprint_Install.label>оружейный верстак (проект)</GunsmithingBench_Blueprint_Install.label>
-
```

Comment:
why is the Gunsmithing bench translation removed?

Reply:
Aren't Frame & Blueprint Defs auto-generated? these seems to be using their origin def's translation, but I'm not sure. Need Russian speakers' help

Reply:
same for "CE_MissinGunsmithingBench". I have a feeling that we have added a bunch of missing tanslations for stuff. Might need to get them and add them to a seperate folder, or perhaps integrate into the Core Language files